### PR TITLE
Add real traffic light detection sample code

### LINKED ISSUE #6: A traffic light detection model for real world
We may not need to train traffic light detection model for real world because there are many publicly available object detection models that can detect traffic lights. https://github.com/tensorflow/models/tree/master/research/object_detection

### PR DESCRIPTION
**Pull request by yaboo-oyabu**
Add real traffic light detection sample code, and update the default TensorFlow version to 1.15.

Figure_1
Figure_2

### CODE REVIEW

**Comment by jotredi** [comment]
I have tried your code with images from the simulator as well. I think we could use it for detect traffic lights from both the simulator and real images.

The model detects near traffic lights pretty well with a confidence cutoff of 0.4

img0

However, is not able to detect traffic lights that are far away from the vehicle (around 300 waypoints) even if I take down the confidence cutoff

img55

but as soon as it approaches the traffic light it starts detecting some of them: 

img76

img88

Even though it doesn't detect traffic lights that are far away, it detects them soon enough for the car to brake smoothly.
For me, we can use this detector :)

**Comment by yaboo-oyabu** [comment]
@jotredi Thanks for the check. Could you kindly approve my pull request? I need someone's approval before the merge.

**Review by jotredi — APPROVED** [review]

File: requirements.txt
Comment (CaoFM) [inline requirements.txt:11]: One of the [project instruction](https://classroom.udacity.com/nanodegrees/nd013/parts/01a340a5-39b5-4202-9f89-d96de8cf17be/modules/702b3c5a-b896-4cca-8a64-dfe0daf09449/lessons/e43b2e6d-6def-4d3a-b332-7a58b847bfa4/concepts/3251f513-2f82-4d5d-88b6-9d646bbd9101) states that:
_Please note that we will not be able to accomodate special launch instructions or run additional scripts from your submission to download files - this includes any changes to the requirements.txt file._

Does that mean reviewer/Carla will not be able to run our model if it is trained under a different version?

Comment (yaboo-oyabu) [reply requirements.txt:11]: @CaoFM Oh... good point.

Looks like this will be a solution: https://stackoverflow.com/questions/53927700/how-to-use-object-detection-api-with-an-old-version-of-tensorflow-v1-3-0

**Pull request merged**
